### ISSUE TITLE
fetch ohlvc issue with $since on GDAX

### ISSUE THREAD

**Issue by stevengroot82** [ccxt/ccxt#1293]
- OS: Linux on Synology DSM
- Programming Language version: PHP 5.6
- CCXT version: 1.10.646
- Exchange: GDAX
- Method: fetch_ohlcv

Yesterday I've tested the fetch_ohlvc function on the GDAX api with start and end times.
Aparently the API cannot handle iso8601-formats including timezones and milliseconds (although this is mentioned in their documentation)
CCXT delivers: `2018-01-01T00:00:00+01:00`
API only accepts `2018-01-01T00:00:00`

Suggested solution is to replace `$this->iso8601` with `$this->YmdHMS`

`public function fetch_ohlcv ($symbol, $timeframe = '1m', $since = null, $limit = null, $params = array ())`
`{`
`    $this->load_markets();`
`    $market = $this->market ($symbol);`
`    $granularity = $this->timeframes[$timeframe];`
`        $request = array (`
`        'id' => $market['id'],`
`        'granularity' => $granularity,`
`    );`
`    $since = $since * 1000;		//Added for conversion of timestamp`
`    if ($since) {`
`        //$request['start'] = $this->iso8601 ($since);`
`        $request['start'] = $this->YmdHMS ($since, 'T');`
`        if (!$limit)`
`            $limit = 200; // max = 200`
`            //$request['end'] = $this->iso8601 ($limit * $granularity * 1000 . $since);`
`            $request['end'] = $this->YmdHMS (($limit * $granularity * 1000)+$since, 'T');`
`        }`
`        $response = $this->publicGetProductsIdCandles (array_merge ($request, $params));`
`        return $this->parse_ohlcvs($response, $market, $timeframe, $since, $limit);`
`    }`

Also as mentioned earlier by issues #1072 and #949 the End-calculation is incorrect and the point (.) should be replaced by the plus (+).

Big thanks for putting this sofisticated piece of code on GitHub!
Really awesome!

Steven

**Comment by kroitor** [comment]
This is fixed in current version 1.10.738. We will be happy if you report back whether it works as expected for you or not. Thx!

**Comment by stevengroot82** [comment]
Hi Kroitor!

Thanks for taking a look at this issue so fast!
I have tested the changes and they work like a charm!

The only thing I had to change was to multiply the timestamp in my code with 1000 before it could be passed correctly to the fetch_OHLCV function as a parameter.

`$lastweek  = mktime(0, 0, 0, date("m")  , date("d")-7, date("Y"));`
`$lastweek = $lastweek * 1000;`
`var_dump ($exchange->fetchOhlcv('ETH/EUR', '1h', $lastweek));`

Best regards,
Steven
